css/style.css

```diff
@@ -1,3 +1,143 @@
-h1 {
-  color: salmon;
+body{
+  background-color: rgb(237,239,236);
+}
+
+/*        Header         */
+.header{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100px;
+}
+.header h1 {
+  color: rgb(134,136,137);
+  font-family: 'Open Sans', sans-serif;
+  margin: 0;
+}
+.header h5{
+  color: rgb(134,136,137);
+  font-family: 'Open Sans', sans-serif;
+  margin: 5px 0;
+}
+
+/* Picture Section One */
+.row2{
+  display: flex;
+  justify-content: center;
+}
+.san1{
+  height: 400px;
+  width: 400px;
+  background-image: url('../img/san-francisco.jpg');
+  color: white;
+  font-family: 'Open Sans', sans-serif;
+  margin: 0 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.ny{
+  height: 400px;
+  width: 400px;
+  background-image: url('../img/new-york.jpg');
+  color: white;
+  font-family: 'Open Sans', sans-serif;
+  margin: 0 60px 0 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
These city classes share a lot of identical key-value pairs. Perhaps you could consolidate those shared properties into a single selector that targets these classes...